README.md

```diff
@@ -66,6 +66,7 @@ In addition to electrs's original configuration options, a few new options are a
 - `--lightmode` - enable light mode (see above)
 - `--cors <origins>` - origins allowed to make cross-site request (optional, defaults to none).
 - `--address-search` - enables the by-prefix address search index.
+- `--index-unspendables` - enables indexing of transactions with provably unspendable outputs.
```

Comment:
This enables the indexing of the unspendable outputs themselves, the transactions containing them would get indexed regardless. You can just remove the "transactions with" part and it'll be good. (here and in the CLI help text too)